frontend/src/app/account/profile/profile.component.html

```diff
@@ -0,0 +1,207 @@
+                        @if (formUtils.isEmailInvalid(editProfileForm)) {
+                            <small class="text-red-300">The provided email is invalid</small>
+                        }
+                    </div>
+                    <div class="form-field">
+                        <div class="flex flex-row align-items-center gap-2">
+                            <i class="pi pi-lock"></i>
+                            <label for="password">Password</label>
+                        </div>
+                        <p-password
+                            inputId="password"
+                            name="password"
+                            class="p-fluid"
+                            formControlName="password"
+                            [toggleMask]="true"
+                            [feedback]="false">
+                        </p-password>
+                    </div>
+                </form>
+
+                <p-button
+                    type="submit"
+                    [label]="isProcessingEdit ? '' : 'Confirm Edit'"
+                    [loading]="isProcessingEdit"
+                    styleClass="w-full justify-content-center mt-4"
+                    [disabled]="isProcessingEdit || !editProfileForm.valid"
+                    (click)="onEditSubmit()">
+                </p-button>
+            </div>
+        </ng-container>
+    } @else if (showEditPassword) {
+        <p-divider layout="vertical" [style]="{ height: '30rem' }"></p-divider>
+
+        <ng-container>
+            <div class="container w-full sm:w-22rem">
+                <h2 class="mt-0 align-self-start">Edit Profile</h2>
+                <form
+                    [formGroup]="editPasswordForm"
+                    (keydown.enter)="onPasswordSubmit()"
+                    (ngSubmit)="onPasswordSubmit()"
+                    class="profile-container">
+                    <div class="form-field">
+                        <div class="flex flex-row align-items-center gap-2">
+                            <i class="pi pi-lock"></i>
+                            <label for="password">Old Password</label>
+                        </div>
+                        <p-password
+                            inputId="password"
+                            name="password"
+                            class="p-fluid"
+                            formControlName="oldPassword"
+                            [toggleMask]="true"
+                            [feedback]="false">
+                        </p-password>
+                    </div>
+
+                    <div class="form-field">
+                        <div class="flex flex-row align-items-center gap-2">
+                            <i class="pi pi-lock"></i>
+                            <label for="password">New Password</label>
+                        </div>
+                        <p-password
+                            inputId="password"
```

Comment:
IDs cannot be resused:
![image](https://github.com/user-attachments/assets/308e0bdb-abec-435a-a69f-46bcd61521e4)

Reply:
Oh right, I also saw this and was trying to chase this down, but it didn't change when I renamed it previously. Seems like after purging the image, making unique ID worked.